sanic/cli/base.py

```diff
@@ -0,0 +1,35 @@
+from argparse import (
+    SUPPRESS,
+    Action,
+    ArgumentParser,
+    RawTextHelpFormatter,
+    _SubParsersAction,
+)
+from typing import Any
+
+
+class SanicArgumentParser(ArgumentParser):
+    def _check_value(self, action: Action, value: Any) -> None:
+        if isinstance(action, SanicSubParsersAction):
+            return
+        super()._check_value(action, value)
+
+
+class SanicHelpFormatter(RawTextHelpFormatter):
+    def add_usage(self, usage, actions, groups, prefix=None):
+        if not usage:
+            usage = SUPPRESS
+            # Add one linebreak, but not two
+            self.add_text("\x1b[1A'")
```

Comment:
I assume this is the equivalent of `\n`? Does a simple `\n` not work?

Reply:
No, it removes a line in the terminal. Argparse is adding `\n\n`, which is quite annoying. It makes a HUGE gap here.

![image](https://user-images.githubusercontent.com/166269/207969559-94983e52-58c0-40cd-a209-7fd604f9aabf.png)

If we do nothing though, it looks bad without a gap. The alternative approach is to just add the padding to the logo itself, but that is not ideal either. Not tied to this approach, and I am not sure how it will work on Windows (other ANSI escape codes seem to work well enough). Care to try?

Reply:
Will give this a try tomorrow!

Reply:
This is how it looks on Windows:
![imagen](https://user-images.githubusercontent.com/25409753/208256894-08e1fa28-d15b-494e-8635-b678b7cdeab0.png)

Reply:
Similar in Windows (CMD):
![imagen](https://user-images.githubusercontent.com/25409753/208256943-2fb02f5b-f8f1-4f36-91ed-6d126bbe8f02.png)

Reply:
perfect :ok_hand: